src/main/java/net/glowstone/GlowServer.java

```diff
@@ -1338,6 +1335,20 @@ public int broadcastMessage(String message) {
     }
 
     @Override
+    public void broadcast(BaseComponent component) {
+        for (Player player : getOnlinePlayers()) {
+            player.spigot().sendMessage(component);
+        }
+    }
+
+    @Override
+    public void broadcast(BaseComponent... components) {
```

Comment:
BaseComponent[] is one message, it is serialized into one string. see BungeeCord code :wink:

Can you squash commits?